x-pack/plugins/dataset_quality/public/components/log_explorer_link.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiLink } from '@elastic/eui';
+import React from 'react';
+import { getRouterLinkProps } from '@kbn/router-utils';
+import {
+  SingleDatasetLocatorParams,
+  SINGLE_DATASET_LOCATOR_ID,
+} from '@kbn/deeplinks-observability';
+import { DataStreamStat } from '../../common/data_streams_stats/data_stream_stat';
+import { useKibanaContextForPlugin } from '../utils';
+
+export const LogExplorerLink = React.memo(
+  ({ dataStreamStat, title }: { dataStreamStat: DataStreamStat; title: string }) => {
+    const {
+      services: { share },
+    } = useKibanaContextForPlugin();
+    const [dataset, namespace] = dataStreamStat.title.split('-');
+    const integration = dataStreamStat.integration?.name;
+
+    const params = {
+      dataset,
+      timeRange: {
+        from: 'now-1d',
+        to: 'now',
+      },
+      integration,
+      filterControls: {
+        namespace: {
+          mode: 'include',
+          values: [namespace],
+        },
+      },
+    } as SingleDatasetLocatorParams;
```

Comment:
**nit:** if something changes in the params data structure, with this type assertion when won't see any error on the current object, move the type definition the variable to type it with the same result:
```ts
const params: SingleDatasetLocatorParams = {
```

Reply:
Addressed in [777c0fd](https://github.com/elastic/kibana/pull/173272/commits/777c0fde3964c5c91dd13790d4df0ce691f1fa52).